bin/hdfcoinc/pycbc_multiifo_combine_statmap

```diff
@@ -0,0 +1,208 @@
+#!/bin/env python
+""" Apply a naive mass binning, assuming that each bin is fully independent, which 
+is a conservative estimate. This clusters to find the most significant foreground, but
+leaves the background triggers alone. """
+
+import h5py, numpy, argparse, logging, pycbc, pycbc.events, lal
+import pycbc.version
+from scipy.special import comb
+import matplotlib as mpl
```

Comment:
Any idea what these imports of plotting code are for ..?

Reply:
Yes - I was going to be doing some plotting as part of my testing but removed it, but forgot to remove the plot imports